### ISSUE TITLE
EC2 module parameter `id` is undocumented.

### ISSUE THREAD

**Issue by z0rc** [ansible/ansible#18969]
_From @z0rc on November 17, 2015 12:38_

The `id` parameter was added at https://github.com/ansible/ansible/pull/2421.

For some reason the parameter's documentation was removed later at https://github.com/ansible/ansible-modules-core/commit/c6b0d469acbb1a1b0508bacaedc5456eb5e9be83#diff-9667dfcde0b7854855c94acb534b156aL31

Can the documentation be restored? I haven't seen notices about deprecation and the functionality is there.

_Copied from original issue: ansible/ansible-modules-core#2481_

**Comment by z0rc** [comment]
_From @robynbergeron on November 25, 2015 17:0_

[module: cloud/amazon/ec2.py]

**Comment by z0rc** [comment]
_From @robynbergeron on November 25, 2015 17:2_

@z0rc Thanks for submitting this bug / docs report.

@tgerla @lwade thanks in advance for taking a look at this issue and for your ongoing work in maintaining this module. Note that this may be just a documentation bug :)

**Comment by z0rc** [comment]
_From @ansibot on August 1, 2016 1:6_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by z0rc** [comment]
_From @ansibot on September 12, 2016 16:59_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by z0rc** [comment]
_From @ansibot on October 6, 2016 15:43_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by z0rc** [comment]
_From @ansibot on October 22, 2016 9:13_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by z0rc** [comment]
_From @ansibot on November 6, 2016 10:3_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by z0rc** [comment]
_From @ansibot on November 22, 2016 10:16_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by z0rc** [comment]
_From @ansibot on December 7, 2016 18:8_

This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
@z0rc Greetings! Thanks for taking the time to open this issue. In order for the community to handle your issue effectively, we need a bit more information. 

Here are the items we could not find in your description:
- issue type
- ansible version
- component name

Please set the description of this issue with this template:
https://raw.githubusercontent.com/ansible/ansible/devel/.github/ISSUE_TEMPLATE.md

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibotdev** [comment]
@z0rc You have not responded to information requests in this issue so we will assume it no longer affects you. If you are still interested in this, please create a new issue with the requested information.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by z0rc** [comment]
For the record this was fixed with https://github.com/ansible/ansible/commit/f6676f495798d6d5ae81e7c9eb3915198ebd85ba.
